src/runtime.rs

```diff
@@ -19,103 +19,130 @@ pub struct Sample {
     /// Maximum parks on any given worker
     pub max_parks: u64,
 
+    pub min_parks: u64,
+
     /// Number of no-op ticks across all workers
     pub num_noops: u64,
 
     /// Maximum number of no-op ticks on any given worker
     pub max_noops: u64,
 
-    /// Number of times a worker stole work
+    pub min_noops: u64,
+
+    /// Number of tasks stolen from other workers
     pub num_steals: u64,
 
-    /// Maximum number of times any given worker stole work
+    /// Maximum number of tasks any given worker stole from other workers
     pub max_steals: u64,
 
+    pub min_steals: u64,
+
+    /// Number of tasks scheduled from **outside** of the runtime
+    pub num_remote_schedules: u64,
+
     /// Number of tasks scheduled locally across all workers
     pub num_local_schedules: u64,
 
-    /// Maximum number of tasks scheduled locally on any given worker.
+    /// Maximum number of tasks scheduled locally on any given worker
     pub max_local_schedules: u64,
 
+    pub min_local_schedules: u64,
```

Comment:
some of these have docs and others don't seems weird

Reply:
going to do a doc pass later.